src/Data/Nat/Base.agda

```diff
@@ -120,11 +129,29 @@ instance
 >-nonZero⁻¹ (suc n) = z<s
 
 ------------------------------------------------------------------------
--- Arithmetic
+-- Another useful (proof-irrelevant) ordering relation
 
 open import Agda.Builtin.Nat public
```

Comment:
These should definitely be opened under the `Arithmetic` header. Maybe it's worth moving them (and `div-helper` and `mod-helper`) before the ordering relations?

Reply:
Hmm... I wondered about this at the time: I need monus for `LessThan`, but not (yet) the other arithmetic. Maybe a separate `import` (but I worried about subsequent `imports` shadowing the renaming of monus...?)

Reply:
Or: restrict the import before `LessThan` to only consist of monus, and move the other arithmetic operations to the subsequent import?